README.md

```diff
@@ -40,27 +40,28 @@ So say we have these chat messages in our Firebase database:
 
 We can represent a chat message with this Java class:
 
-    public class ChatMessage {
-        String message;
-        String name;
+```java
```

Comment:
Nice! I know about this GFMD feature, but somehow am too used to StackOverflow Markdown.